doc/sphinx/conf.py

```diff
@@ -47,7 +47,7 @@
 # -- General configuration ------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
-#needs_sphinx = '1.0'
+needs_sphinx = '1.7.8'
```

Comment:
This value is now written twice (here and in the README). How can we ensure they stay consistent?

Reply:
Three times actually, you forgot the Dockerfile. So the problem is not new, and is in fact also present with other dependencies, such as OCaml itself. We should think of a general solution.